src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,43 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected LocalDateTime at;
+    private DateTimeFormatter displayFormatter = DateTimeFormatter.ofPattern("dd MMM uuuu HHmm");
+    private DateTimeFormatter saveFormatter = DateTimeFormatter.ofPattern("dd/MM/uuuu HHmm");
+
+    /**
+     * Constructor
+     *
+     * @param description task description
+     * @param at starts at
+     */
+    public Event(String description, LocalDateTime at) {
+        super(description, false);
+        this.at = at;
+    }
+
+    /**
+     * Constructor for when reading file from storage
+     *
+     * @param description task description
+     * @param isDone check complete status
+     * @param at starts at
+     */
+    public Event(String description, String isDone, LocalDateTime at) {
+        super(description, isDone == "1");
+        this.at = at;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (at: " + at.format(displayFormatter) + ")";
+    }
+
+    @Override
+    public String toSaveString() {
+        return "E|" + super.toSaveString() + "|" + at.format(saveFormatter);
```

Comment:
I like how you used another string formatter for your tasks when saving to storage, allowing for easier readability when re-loading.